lib/request_cache/plug.ex

```diff
@@ -140,7 +140,17 @@ defmodule RequestCache.Plug do
   end
 
   defp request_cache_labels(conn) do
-    conn_request(conn)[:labels]
+    with {:ok, query, _variables} <- Util.fetch_query(conn),
+         {:ok, query_name} <- Util.extract_query_name(query) do
+      query_name
+      |> Macro.underscore()
+      |> String.to_atom()
+      |> List.wrap()
+    else
+      {:query_not_found} -> [:query_not_found]
+      {:query_name_not_found} -> [:query_name_not_found]
```

Comment:
How does this happen, this return format is super non standard. Can you wrap it in ok or error?

Reply:
It's an extension of `{:not_found}`, but easy enough to make `{:not_found, :query}` / `{:not_found, :query_name}` or `{:error, :query_not_found}` / `{:error, :query_name_not_found}`.